src/main/java/org/threadly/litesockets/Client.java

```diff
@@ -372,12 +390,13 @@ public void setReader(final Reader reader) {
    * and network are keeping up with the data going in/out.</p>
    * 
    * @param size max buffer size in bytes.
+   * @deprecated use the {@link #clientOptions()} and set the {@link ClientOptions#setMaxClientReadBuffer(int size)} call.
    */
+  @Deprecated
   public void setMaxBufferSize(final int size) {
-    ArgumentVerifier.assertGreaterThanZero(size, "size");
-    maxBufferSize = size;
+    clientOptions().setMaxClientReadBuffer(size);
```

Comment:
this is just an internal int, so it will set assuming it passed the >0 check, other wise it will throw.